code/modules/mob/living/carbon/human/species_types/zombies.dm

```diff
@@ -33,10 +38,13 @@
 	var/regen_cooldown = 0
 	changesource_flags = MIRROR_BADMIN | WABBAJACK | ERT_SPAWN
 
+// Zombies do not stabilize body tempurature they are the walking dead and are cold blooded
+/datum/species/zombie/natural_bodytemperature_stabilization(mob/living/carbon/human/H)
+	return 0

```

Comment:
same thing with zombies- do they now just die after being set on fire for long enough to be heated up to a damaging temperature range, even after extinguishing themselves (since they'll still be hot)?

Reply:
same as before environment cools them down and acts faster then natural.

Reply:
I could also just stop them from taking damage from high body temperature. but so far all my tests zombies and lizards stabilize once put out.